app/controllers/admin_controller.rb

```diff
@@ -1,7 +1,6 @@
 class AdminController < ApplicationController
 
   def index
-    @merchants = Merchant.all
-    @invoices = Invoice.all
+    @top_customers = Customer.all_successful_transactions_count.order('successful_transactions_count desc').limit(5)
```

Comment:
This might be too much model-ish stuff to do in the controller. Perhaps we could note and come back to this later if my thought is correct and there's time to refactor.